README.md

```diff
@@ -1,70 +1,38 @@
-# Getting Started with Create React App
+# Weather App
```

Comment:
This is a nice basic README, but it is lacking in a few things. Some things that would make it much better:
- what technologies are used,
- screenshots of the app in the browser (mobile and desktop screen sizes especially if the app is responsive),
- a bit more detail on what the app actually does, ie does it work for all cities in all countries? What date range does it show?
- what would you do if you had more time?
- your name!

If you want to play around with markdown to make the formatting of this a bit more interesting, have a look at this cheatsheet [guide to markdown](https://www.markdownguide.org/cheat-sheet/). Also useful is [dillinger.io](https://dillinger.io/) so you can play with what the file will look like without having to do multiple commits to the repo to see the results